plugins/linux-pipewire/pipewire.c

```diff
@@ -1136,7 +1134,7 @@ static void start(obs_pipewire_data *obs_pw)
 
 	new_request_path(obs_pw, &request_path, &request_token);
 
-	blog(LOG_INFO, "[pipewire] asking for %s…",
+	blog(LOG_INFO, "[pipewire] Asking for %s…",
```

Comment:
Replace the special character `…` with `...` (three separate `.` characters).

Reply:
Ellipses are no more.